simpleline/render/io_manager.py

```diff
@@ -113,39 +117,52 @@ def draw(self, active_screen):
         except Exception:    # pylint: disable=broad-except
             self._event_loop.enqueue_signal(ExceptionSignal(self))
 
-    def process_input(self, active_screen):
+    def process_input(self, active_screen, key):
         """Process input from the screens.
 
         Result of the processing is saved in the `processed_result` property.
 
         :param active_screen: Screen demanding this input processing.
         :type active_screen: Classed based on `simpleline.render.screen.UIScreen`.
 
+        :param key: Key from the user input.
```

Comment:
Is it just one character from the user input?

Reply:
Could be more, separator is Enter key here. Everywhere it was saved in variable key that I somehow get used to it. I'll fix this.